q2_alignment/plugin_setup.py

```diff
@@ -33,7 +36,17 @@
     parameters={'max_gap_frequency': Float,
                 'min_conservation': Float},
     outputs=[('masked_alignment', FeatureData[AlignedSequence])],
+    input_descriptions={'alignment': 'The alignment to be masked.'},
+    parameter_descriptions={
+        'max_gap_frequency': ('The maximum fraction of sequences that can '
+                              'contain a gap character in a specific column '
+                              'for that column to be retained.'),
+        'min_conservation': ('The minimum fraction of sequences that must '
```

Comment:
Is this description accurate? Maybe it's better to indicate conservation is calculated using "Inverse Shannon Uncertainty", and link to the scikit-bio docs or a more appropriate reference? I'm having a hard time interpreting the current description.